apps/mobile-wallet/src/components/footers/FooterMenu.tsx

```diff
@@ -48,18 +52,26 @@ const FooterMenu = ({ state, descriptors, navigation, style }: FooterMenuProps)
     </>
   )
 
+  const handleFooterLayout = (e: LayoutChangeEvent) => {
+    setFooterHeight(e.nativeEvent.layout.height + insets.bottom)
+  }
+
   return (
-    <View style={[style]}>
-      {Platform.OS === 'ios' ? (
-        <>
-          <FooterMenuContrastedBackground />
-          <FooterMenuContentBlured tint={theme.name} intensity={50} style={{ paddingBottom: insets.bottom }}>
-            {footerContent}
-          </FooterMenuContentBlured>
-        </>
-      ) : (
-        <FooterMenuContent style={{ paddingBottom: insets.bottom }}>{footerContent}</FooterMenuContent>
-      )}
+    <View style={[style]} onLayout={handleFooterLayout}>
```

Comment:
I know you didn't change this code but is wrapping in an array needed?

Reply:
It's not. Fixed.

Reply:
Maybe you forgot to push? I can still see it wrapped in array

Reply:
Sry didn't know you were working today, so I didn't push yet :P Pushed now.